test/evergreen/install_gcp_dependencies.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
```

Comment:
Can we place a comment here explaining this bash script does? And the future work entailed to make this better and why we are doing like this?

Reply:
updated: e33a0155f8f59f6938296201e55a2f7e401a7f38